sample-input/benchmarks/takeda/model_1/takeda-rodded.py

```diff
@@ -24,7 +24,8 @@
 a = universes['Control Rod']
 r = universes['Reflector']
 
-lattices['Root'].setWidth(width_x=5.0/refines, width_y=5.0/refines, width_z=5.0/refines)
+lattices['Root'].setWidth(width_x=5.0/refines, width_y=5.0/refines,
+                          width_z=5.0/refines)
```

Comment:
No need to split this line unless the `setUniverses()` call is also shortened.

Splitting after the parenthesis makes the longest line is 79 chars:

```python
lattices['Root'].setUniverses(
    [[np.repeat([r, r, r, r, r], refines).tolist()] * 4 * refines +
     [np.repeat([r, r, r, a, r], refines).tolist()] * refines] * 2 * refines +
    [[np.repeat([r, r, r, r, r], refines).tolist()] * 2 * refines +
     [np.repeat([c, c, c, r, r], refines).tolist()] * 2 * refines +
     [np.repeat([c, c, c, a, r], refines).tolist()] * refines] * 3 * refines)
```

Reply:
ok fixing